.env.example

```diff
@@ -34,9 +34,9 @@ LOGS_ACTIVE=false
 GITHUB_TOKEN=
 
 #ifps
-IPFS_PROJECT_ID=
```

Comment:
this project _does not_ use `NextJs` so environment variables do not need `NEXT_` prefix